style.css

```diff
@@ -0,0 +1,118 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;600;700&display=swap");
+
+:root {
+  --color-darkblue: #1b1b32;
+}
+
+*,
+*::before,
+*::after {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+body {
+  font-family: poppins;
+}
+
+#overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  background-color: rgba(58, 58, 158, 0.6);
+  z-index: 1;
+}
+main {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: url(https://images.pexels.com/photos/273671/pexels-photo-273671.jpeg?auto=compress&cs=tinysrgb&w=600);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 100%;
+  width: 100%;
+}
```

Comment:
Kindly The background image is not well visible like it shows in the Figma design to fix that I can suggest you adjust it a bit position you can do it by using min-height: calc(100vh - 599px); 👍

Reply:
You did really well adding more inline comments and giving a polite review as suggested by the previous reviewer, please as a part of a good communication structure in reviews, you are to give a reason `why` you are suggesting such changes from the student to enable them to know the importance of them implementing the changes you requested for. You did that in your first review, please include it in your new reviews 😄